src/tokenizer.ts

```diff
@@ -182,6 +185,12 @@ export class Tokenizer {
         this.tokens.push(new Token(type, lexeme, line, col, this.current - lexeme.length))
     }
 
+    private addMultiLineStringToken(type: TokenType) {
+        const line = this.line
+        const col = this.col;
+        const lexeme = this.source.slice(this.start + 3, this.current - 3);
```

Comment:
Might want to add a comment explaining why +3 and -3 here. I understand that it's due to the `"""` but that's not always obvious to others :).